public/assets/js/scripts.js

```diff
@@ -285,4 +285,27 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    let periodic = $('.periodic-investment').val();
```

Comment:
Kindly update this (and others below) to `const` not `let` since we are not updating their values later in the code.
```suggestion
    const periodic = $('.periodic-investment').val();
```